assets/l10n/app_de.arb

```
@@ -0,0 +1,474 @@
+  "tagCocktail": "Cocktail",
+  "tagTropicalCocktail": "Tropischer Cocktail",
+  "tagMateTea": "Mate-Tee",
+  "tagMilk": "Milch",
+  "tagKamaboko": "Kamaboko",
+  "tagOden": "Oden",
+  "tagCheese": "Käse",
+  "tagEgg": "Ei",
+  "tagFriedEgg": "Spiegelei",
+  "tagButter": "Butter",
+  "done": "Fertig",
+  "cancel": "Abbrechen",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)